test/integration/dialects/postgres/dao.test.js

```diff
@@ -332,6 +332,31 @@ if (dialect.match(/^postgres/)) {
         });
       });
 
+      it('should not rename hstore fields', function() {
```

Comment:
No can do... It is the Model that transforms the where object into the stringified where conditions. Tests here will fail with 'Error: Invalid value [object Object]'